front/src/pages/companies/companies-columns.tsx

```diff
@@ -65,6 +65,11 @@ export const useCompaniesColumns = () => {
               company.name = value;
               updateCompany(company);
             }}
+            chipClickHandler={(editing) => {
```

Comment:
Yes, I feel the api should be onChange and onRemove here

Reply:
maybe we can also imagine that we only have onChange here and value can be string or null

Reply:
It actually looks that it's what you've done for account Owner below

Reply:
Actually (3rd one :D), why do we have this on company name? it's not a relation!

Reply:
remove handler here